event/listener.php

```diff
@@ -78,7 +78,6 @@ static public function getSubscribedEvents()
 
 			// Message parsing
 			'core.modify_text_for_display_before'			=> 'replace_mentions_for_display',
-			'core.modify_text_for_display_before'			=> 'replace_mentions_for_display',
```

Comment:
`core.modify_text_for_display_before` is the one that's already in there. You might be thinking about `core.modify_format_display_text_after`, although I'm not sure it applies to your extension.